sdk/aqueduct/integrations/s3_integration.py

```diff
@@ -66,7 +73,9 @@ def file(
                             extract=ExtractSpec(
                                 service=integration_info.service,
                                 integration_id=integration_info.id,
-                                parameters=S3ExtractParams(filepath=filepath, format=format),
+                                parameters=S3ExtractParams(
+                                    filepath=json.dumps(filepath), format=format
```

Comment:
Can we document this somewhere that filepath needs to be JSON serialized

Reply:
documented.